IABTechLab/uid2-self-serve-portal
# Add ability to rename and delete key pairs

- Key pair name can be edited via edit button under Actions
- Key pair can be deleted ("disabled" on our end) via trash can button under Actions
- Disabled is no longer a property when a user creates a new key pair
- Disabled key pairs are filtered out in the api instead of the front end
- Removed display of disabled key in UI 
- Created storybook components for KeyPairDisableDialog and KeyPairEditDialog

Test Plan:

- Edit existing key pair
- Delete existing key pair
- Create new key pair and edit, then disable it
- Delete a key pair and then create a new key pair with the same name and make sure it can also be created
- Check that creating a key pair does not show a disabled checkbox
